index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Technical Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header><a href="#introduction" class="nav-link">Introduction</a></header>
+        <header><a href="#why-you" class="nav-link">What you should already know</a></header>
+        <header><a href="#avascript" class="nav-link">JavaScript and Java</a></header>
+        <header><a href="#hello-world" class="nav-link">Hello world</a></header>
+        <header><a href="#variables" class="nav-link">Variables</a></header>
+    </nav>

```

Comment:
There should be only one header tag in your project.
You can put the nav tag inside the header tag.